lm_eval/models/huggingface.py

```diff
@@ -570,6 +573,25 @@ def _create_model(
             self._model = PeftModel.from_pretrained(
                 self._model, peft, revision=revision
             )
+        elif delta:
```

Comment:
Is it a safe assumption that one will never pass `peft` and `delta` at the same time? 

I think instead of doing an `elif` here, having an assert / error raised if both `peft` and `delta` are not none would be helpful.